dev/PipsControl/TestUI/PipsControlPage.xaml

```diff
@@ -0,0 +1,19 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<local:TestPage
+    x:Class="MUXControlsTestApp.PipsControlPage"
+    x:Name="PipsControlTestPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:MUXControlsTestApp"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid  Margin="12" Padding="50">
+        <StackPanel Orientation="Vertical">
+            <muxc:PipsControl x:Name="pipsControl" NumberOfPages="30" MaxDisplayedPages="10" AutomationProperties.Name="TestPipsControl"
```

Comment:
Can you make the test UI end to end ? wire it up to a collection control like ListView or Repeater ?

Reply:
Actually, we should add a small sample for FlipView as well, which would be a common usage.